### DISCUSSION TITLE
aws-actions/configure-aws-credentials@v1 and aws-actions/amazon-ecr-login@v1 are not allowed to be used

### DISCUSSION THREAD

**Discussion by yawei-workmate** [community/community#57508] [Actions]
### Select Topic Area

Bug

### Body

Hi There,

On 8th-Jun around 12pm(+8), our build start to facing 
aws-actions/configure-aws-credentials@v1 and aws-actions/amazon-ecr-login@v1 are not allowed to be used in xxx. Actions in this workflow must be: within a repository owned by xxx or created by GitHub." error, we didnt update any env and github workflow related thing, may I check if anything happening in github side? 

Below are my steps
 steps:
      - name: Check out the repo
        uses: actions/checkout@v3
      - name: Configure AWS credentials
        uses: aws-actions/configure-aws-credentials@v1
        with:
          aws-access-key-id: ${{ secrets.aws_access_key_id }}
          aws-secret-access-key: ${{ secrets.aws_secret_access_key }}
          aws-region: ${{ secrets.aws_default_region }}
      - name: Login to Amazon ECR
        id: login_ecr
        uses: aws-actions/amazon-ecr-login@v1

**Comment by airtower-luna** [comment]
> aws-actions/configure-aws-credentials@v1 and aws-actions/amazon-ecr-login@v1 are not allowed to be used in xxx.

I assume "xxx" is an organization? In that case an organization admin probably introduced limits on which actions can run, see https://docs.github.com/en/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization. Best talk to them and ask them to allow the actions you need.

**Reply by yawei-workmate** [reply]
thanks, is due to this action permission issue, didnt aware someone else from organization updated the permission

**Reply by airtower-luna** [reply]
Glad I could help! Please don't forget to mark the answer. :slightly_smiling_face:
